math/vecops/inc/ROOT/RVec.hxx

```diff
@@ -1475,6 +1505,10 @@ public:
    }
 
    using SuperClass::at;
+
+   friend bool ROOT::Detail::VecOps::IsSmall<T>(const RVec &v);
+
+   friend bool ROOT::Detail::VecOps::IsAdopting<T>(const RVec &v);
```

Comment:
Try with:
```suggestion
   friend bool ROOT::Detail::VecOps::IsSmall<T>(const RVec<T> &v);

   friend bool ROOT::Detail::VecOps::IsAdopting<T>(const RVec<T> &v);
```

Reply:
Change applied, and waiting for the build now. Very interesting catch!